kymatio/__init__.py

```diff
@@ -2,18 +2,23 @@
 ### Snippet copied from sklearn.__init__
 import warnings
 import re
+
 warnings.filterwarnings('always', category=DeprecationWarning,
                         module=r'^{0}.*'.format(re.escape(__name__)))
 ### End Snippet
 
 __all__ = [
             'Scattering1D',
-            'Scattering2D',
+            'Scattering2DTorch',
```

Comment:
Would set `__all__` after trying to import stuff and then fill it with what is available